File: Python/Human.py
class Human():
    __NIK = ""
    __nama = ""
    __jenis_kelamin = ""

    def __init__(self, NIK="", nama="", jenis_kelamin=""):
        self.__NIK = NIK
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin

    def getNIK(self):
        return self.__NIK

    def getnama(self):
        return self.__nama

    def getjenis_kelamin(self):
        return self.__jenis_kelamin

    def setNIK(self, NIK):
        self.__NIK = NIK

    def setnama(self, nama):
        self.__nama = nama

    def setjenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

File: PHP/program/Human.php
<?php
class Human
{
    private $NIK;
    private $nama;
    private $jenis_kelamin;

    public function __construct($NIK = "", $nama = "", $jenis_kelamin = "")
    {
        $this->NIK = $NIK;
        $this->nama = $nama;
        $this->jenis_kelamin = $jenis_kelamin;
    }

    public function setNIK($NIK)
    {
        $this->NIK = $NIK;
    }
    public function setNama($nama)
    {
        $this->nama = $nama;
    }
    public function setjenis_kelamin($jenis_kelamin)
    {
        $this->jenis_kelamin = $jenis_kelamin;
    }
    public function getNIK()
    {
        return $this->NIK;
    }
    public function getNama()
    {
        return $this->nama;
    }
    public function getjenis_kelamin()
    {
        return $this->jenis_kelamin;
    }
}
